UnityProject/Assets/Scripts/Electricity/Wire/MachineConnectorSpriteHandler.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+public class MachineConnectorSpriteHandler : NetworkBehaviour
+{
+	public char InNorth = '0';
+	public char InSouth = '0';
+	public char InWest = '0';
+	public char InEast = '0';
+
+	public SpriteRenderer SRNorth;
+	public SpriteRenderer SRSouth;
+	public SpriteRenderer SRWest;
+	public SpriteRenderer SREast;
+
+	[SyncVar(hook = "UpdateSprites")]
+	public string Syncstring;
+
+
+	public WireConnect Wire;
+	public void Check()
+	{
+		InNorth = '0';
+		InSouth = '0';
+		InWest = '0';
+		InEast = '0';
+		Syncstring = "0000";
+		HashSet<ElectricalOIinheritance> connections = new HashSet<ElectricalOIinheritance>();
+		connections = ElectricityFunctions.SwitchCaseConnections(Wire.transform.localPosition, Wire.matrix, Wire.InData.CanConnectTo, Connection.MachineConnect, Wire);
```

Comment:
what is `Wire.transform.localPosition` used here for? it might not be accurate

Reply:
It's used for accessing the location on the matrix  of itself , the reason I use it is because it's the same in find connections